javav2/example_code/transcribe/src/main/java/com/amazonaws/transcribestreaming/StreamTranscriptionBehavior.java

```diff
@@ -1,66 +1,64 @@
- * permissions and limitations under the License.
- */
-package com.amazonaws.transcribestreaming.retryclient;
-
-import software.amazon.awssdk.services.transcribestreaming.model.StartStreamTranscriptionResponse;
-import software.amazon.awssdk.services.transcribestreaming.model.TranscriptResultStream;
-
-/**
- * Defines how a stream response should be handled.
- * You should build a class implementing this interface to define the behavior.
- */
-public interface StreamTranscriptionBehavior {
-    /**
-     * Defines how to respond when encountering an error on the stream transcription.
-     *
-     * @param e The exception
-     */
-    void onError(Throwable e);
-
-    /**
-     * Defines how to respond to the Transcript result stream.
-     *
-     * @param e The TranscriptResultStream event
-     */
-    void onStream(TranscriptResultStream e);
-
-    /**
-     * Defines what to do on initiating a stream connection with the service.
-     *
-     * @param r StartStreamTranscriptionResponse
-     */
-    void onResponse(StartStreamTranscriptionResponse r);
-
-
-    /**
-     * Defines what to do on stream completion
-     */
-    void onComplete();
-}
-// snippet-end:[transcribe.java-streaming-client-behavior]
\ No newline at end of file
+// snippet-sourcedescription:[StreamTranscriptionBehavior.java is an interface that you implement for streaming transcription.]
+// snippet-service:[transcribe]
+// snippet-keyword:[Java]
+// snippet-sourcesyntax:[java]
+// snippet-keyword:[Amazon Transcribe]
+// snippet-keyword:[Code Sample]
+// snippet-keyword:[TranscribeStreamingAsyncClient]
+// snippet-keyword:[StartStreamTranscriptionResponse]
+// snippet-sourcetype:[snippet]
+// snippet-sourcedate:[2019-01-10]
+// snippet-sourceauthor:[AWS]
+// snippet-start:[transcribe.java-streaming-client-behavior]
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ */
+package com.amazonaws.transcribestreaming;
+
+import software.amazon.awssdk.services.transcribestreaming.model.StartStreamTranscriptionResponse;
+import software.amazon.awssdk.services.transcribestreaming.model.TranscriptResultStream;
+
+/**
+ * Defines how a stream response should be handled.
+ * You should build a class implementing this interface to define the behavior.
+ */
+public interface StreamTranscriptionBehavior {
+    /**
+     * Defines how to respond when encountering an error on the stream transcription.
+     *
+     * @param e The exception

```

Comment:
For parameter docs, I'd either say something more meaningful or remove them.

Reply:
fixed